content/_data/changelogs/lts.yml

```diff
@@ -9733,6 +9733,526 @@
+  - type: bug
+    category: bug
+    pull: 8485
+    issue: 71890
+    authors:
+      - benebsiny
+    pr_title: "[JENKINS-71890] Model links in build history views and administrative\
+      \ monitors no longer work"
+    message: |-
+      Restore context menus of model links in build history views and in administrative monitors.
+  - type: bug
+    category: bug
+    pull: 8491
+    issue: 72018
+    authors:
+      - flabrie
+    pr_title: "[JENKINS-72018] Form entries in .repeated-chunk using .show-if-not-only\
+      \ CSS class are not hidden when they are alone"
+    message: |-
+      Hide the delete button from the only repeatable element in configuration forms when at least one element is expected.
+  - type: bug
+    category: bug
+    pull: 8490
+    issue: 72016
+    authors:
+      - jglick
+    pr_title: "[JENKINS-72016] Define a thread pool for `ProxyConfiguration`'s\
+      \ `HttpClient`"
+    message: |-
+      Do not create a large number of threads when making numerous HTTP requests.
+  - type: bug
+    category: bug
+    pull: 8471
+    issue: 71983
+    authors:
+      - mawinter69
+    pr_title: "[JENKINS-71983] allow symbols in MenuItem#withIconClass"
+    message: |-
+      Symbols display in breadcrumbs now.
+  - type: bug
+    category: bug
+    pull: 8416
+    issue: 38520
+    authors:
+      - dependabot[bot]
+    pr_title: "[JENKINS-38520] Every message in agent log appears twice with the\
+      \ `-agentLog` option"
+    message: |-
+      Message no longer appears twice when the <code>agentLog</code> option is used.
+  - type: bug
+    category: bug
+    pull: 8388
+    issue: 71848
+    authors:
+      - daniel-beck
+    pr_title: "[JENKINS-71848] Remove admin monitors popup from `/manage/` again"
+    message: |-
+      Hide administrative monitors icons/popup in the header of <strong>Manage Jenkins</strong>, as they're shown directly on the page.
+  - type: bug
+    category: bug
+    pull: 8378
+    issue: 61452
+    authors:
+      - jglick
+    pr_title: "[JENKINS-61452] Tolerate corrupt Base64 in `PlainTextConsoleOutputStream`"
+    message: |-
+      The plain text console log will still be printed even if some console annotations are corrupt.
+  - type: bug
+    category: bug
+    pull: 8387
+    issue: 71833
+    authors:
+      - daniel-beck
+    pr_title: "[JENKINS-71833] Fix link to job in slow trigger admin monitor"
+    message: |-
+      Fix link to job in the message informing administrators of trigger computations that run for an unusually long time.
+  - type: bug
+    category: bug
+    pull: 8089
+    authors:
+      - dten
+    pr_title: Use standard size node icon even with long node names
+    message: |-
+      Use standard size node icon even with long node names.
+  - type: bug
+    category: bug
+    pull: 8322
+    issue: 71744
+    authors:
+      - mawinter69
+    pr_title: "[JENKINS-71744] Fix context menu"
+    message: |-
+      Fix "Manage Jenkins" context menu.
```

Comment:
This bug was never visible to LTS users, so it does not need to be in the changelog  Refer to JENKINS-71744 for the details.
```suggestion
```